src/Revise.jl

```diff
@@ -440,11 +440,13 @@ function eval_new!(exs_sigs_new::ExprsSigs, exs_sigs_old, mod::Module)
                     if VERSION < v"1.3.0" || !isexpr(thunk, :thunk)
                         thunk = ex
                     end
-                    for p in workers()
-                        p == myid() && continue
-                        try   # don't error if `mod` isn't defined on the worker
-                            remotecall(Core.eval, p, mod, thunk)
-                        catch
+                    if myid() == 1
```

Comment:
`myid() == 1` used to be implied by the check `myid() == 1 && return nothing` in `__init__()` - other processes would never run this code